config.json

```diff
@@ -3,20 +3,27 @@
     "auth": {
       "open_signup": true,
       "require_email": true
-    }
   },
+  "ui": true
+  },
+  "logscope":{},
   "teraserver": {
     "shutdown_timeout": 5,
     "plugins": {
       "names": [
-        "teranaut"
+        "teranaut", "logscope"
       ],
-      "path": "/Users/jarednoble/Projects/terascope/teraserver/plugins"
+      "path": "/Users/ciorg/Terascope/teraserver/plugins"
+  },
+    "stats": {
+      "es_onnection": "default",
```

Comment:
`es_onnection`?

Also please convert this example config to YAML and completely replace the JSON version.